test/functional/sqlsrv/MsCommon.inc

```diff
@@ -84,6 +84,12 @@ function isDaasMode()
     return ($daasMode ? true : false);
 }
 
+function isLocaleDisabled()
+{
+    global $daasMode, $localeDisabled;
+    return ($daasMode || $localeDisabled);
+}
+
```

Comment:
We don't use isDaasMode anymore now right? We can remove it

Reply:
We still use it in specific tests for Azure